packages/mdc-select/mdc-select.scss

```diff
@@ -147,11 +138,8 @@ $mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timin
     transform: scaleY(1);
     transform-origin: bottom;
     transition: $mdc-select-menu-transition;
-    background-color: rgba(black, .5);
 
     &::after {
```

Comment:
I dont see any ::after in the mixins...did we lose this logic?

Reply:
On focus/open, the bottom-line turned the primary color. Then the ::after psuedo element applied a transition from the center moving outward that is also the primary color. 

Purple is the surface:focus .mdc-select__bottom-line element. 
Green is the ::after psudo element animating outward. 

Removing the entire ::after element felt like a separate PR. 

<img width="634" alt="screen shot 2018-01-11 at 3 59 04 pm" src="https://user-images.githubusercontent.com/34519388/34853407-65cddf92-f6e8-11e7-9e6b-28a2075d8f4b.png">